src/sentry/static/sentry/app/utils/fileExtension.tsx

```diff
@@ -33,7 +33,11 @@ const FILE_EXTENSION_TO_PLATFORM = {
  */
 export function getFileExtension(fileName: string): string | undefined {
   // this won't work for something like .spec.jsx
-  return fileName.split('.').pop();
+  const segments = fileName.split('.');
```

Comment:
Which filename wasn't working for dart?

Reply:
This is handling the edge-case where there is no file extension at all.